spec/api/v1/requests/profile_request_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe "User Profile" do
+  it "gets a user's profile" do
+
+    user = create(:user)
+
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+
+    get "/api/v1/profile", params: { "id" => "#{user.id}" }.to_json, headers: { 'CONTENT_TYPE' => 'application/json', 'ACCEPT' => 'application/json' }
```

Comment:
This looks fine for now since we're hard-coding the user login (no authentication)